app/concerns/cmr_collections_helper.rb

```diff
@@ -27,23 +27,26 @@ def get_revisions(concept_id, revision_id)
     revisions
   end
 
-  def get_associated_services(response)
-    if response.present?
-      service_ids = response.dig(0, 'meta', 'associations', 'services')
-      # This can happen when there are variable associations and probably tool
-      # in the future.
-      @services = [] and return unless service_ids
+  def get_associated_concepts(response, concepts = ['tools', 'services'])
```

Comment:
in the controllers you actually pass in `collection_response.body['items']` and `@revisions`, so I feel like it shouldn't be called `response` here to avoid any potential future confusion